lib/net/http/response.rb

```diff
@@ -290,7 +290,7 @@ def read_body_0(dest)
 
       clen = content_length()
       if clen
-        @socket.read clen, dest, true   # ignore EOF
+        @socket.read clen, dest
```

Comment:
I'm assuming `@socket.read` semantics are as described in https://ruby-doc.org/core-2.5.0/IO.html#method-i-read, specifically "Note that this method behaves like the fread() function in C"

Reply:
Uh, `#read` doesn't accept 3 arguments? I'm a bit confused.

Reply:
It ends up calling [`Net::BufferedIO#read`](https://github.com/ruby/ruby/blob/v2_5_3/lib/net/protocol.rb#L117) which accepts the optional `ignore_eof` parameter.